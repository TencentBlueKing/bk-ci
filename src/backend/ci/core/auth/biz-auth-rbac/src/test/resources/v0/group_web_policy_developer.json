{
  "type": "group_web_policy",
  "project_id": "mht",
  "subject": {
    "type": "group",
    "id": 4,
    "name": "开发人员"
  },
  "permissions": [
    {
      "actions": [
        {
          "id": "pipeline_create"
        }
      ],
      "resources": [
        {
          "type": "pipeline",
          "paths": [

          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "credential_create"
        }
      ],
      "resources": [
        {
          "type": "credential",
          "paths": [

          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "cert_create"
        }
      ],
      "resources": [
        {
          "type": "cert",
          "paths": [

          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "repertory_create"
        }
      ],
      "resources": [
        {
          "type": "repertory",
          "paths": [

          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "environment_create"
        }
      ],
      "resources": [
        {
          "type": "environment",
          "paths": [

          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "env_node_create"
        }
      ],
      "resources": [
        {
          "type": "env_node",
          "paths": [

          ]
        }
      ]
    }
  ],
  "members": [
    {
      "type": "user",
      "id": "5656"
    }
  ]
}
