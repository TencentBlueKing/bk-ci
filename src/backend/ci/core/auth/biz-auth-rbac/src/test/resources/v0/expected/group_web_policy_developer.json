[ {
  "system" : "bk_ci_rbac",
  "actions" : [ {
    "id" : "pipeline_create"
  }, {
    "id" : "credential_create"
  }, {
    "id" : "cert_create"
  }, {
    "id" : "repertory_create"
  }, {
    "id" : "environment_create"
  }, {
    "id" : "env_node_create"
  } ],
  "resources" : [ {
    "system" : "bk_ci_rbac",
    "type" : "project",
    "paths" : [ [ {
      "system" : "bk_ci_rbac",
      "type" : "project",
      "id" : "bkdevops",
      "name" : "bkdevops"
    } ] ]
  } ]
} ]
